src/bridges/multitoken/ExampleBridge.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright 2020 Spilsbury Holdings Ltd
+pragma solidity >=0.6.10 <=0.8.10;
+pragma experimental ABIEncoderV2;
+
+import {SafeMath} from '@openzeppelin/contracts/utils/math/SafeMath.sol';
+
+import {IDefiBridge} from '../../interfaces/IDefiBridge.sol';
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+import {AztecTypes} from '../../aztec/AztecTypes.sol';
+
+contract MutliBridgeContract is IDefiBridge {
```

Comment:
What is this contract here for? It looks just like some forgotten test file. If that's the case, please remove it.